kolibri/plugins/coach/assets/src/modules/classSummary/dataHelpers.js

```diff
@@ -77,8 +77,16 @@ export default {
    */
   getRecipientNamesForExam(state) {
     return function(exam) {
+      const adHocLearners = this.getAdHocLearners(exam.assignments).map(
```

Comment:
Hrm - I think for safety the anonymous function defined here needs to be an arrow function `exam => {` - otherwise there is a risk that `this` will not be preserved in later invocations.

_This_ is probably the case for the other getters that return functions above too.